docs/guide/syntax.md

```diff
@@ -64,15 +64,15 @@ Refer to [customization](/custom/) for more details.
 
 ## Code Snippet
 
-A big reason I am making this is that I need to make my code looks just right in the slides. So just as you expected, you can use Markdown favored code block to highlight your code.
+A big reason I am building Slidev is needing to make my code look just right in the slides. So just as you expected, you can use Markdown flavored code block to highlight your code.
 
-~~~ts
+~~~md
```

Comment:
Ah, this is on purpose to fool Prism to get proper highlighting, Can you revert them all? Thanks.

Reply:
sure thing